cmd/ferretdb/main.go

```diff
@@ -196,6 +196,7 @@ func setupLogger(stateProvider *state.Provider) *zap.Logger {
 		zap.String("package", info.Package),
 		zap.Bool("debugBuild", info.DebugBuild),
 		zap.Any("buildEnvironment", info.BuildEnvironment.Map()),
+		zap.Bool("disable-pushdown", cli.Test.DisablePushdown),
```

Comment:
```suggestion
		zap.Bool("disablePushdown", cli.Test.DisablePushdown),
```

Note that it uses camelCase